lisa/sut_orchestrator/azure/tools.py

```diff
@@ -73,31 +73,31 @@ def deprovision(self) -> None:
         # self.run("-deprovision+user --force", sudo=True)
         self.run("-deprovision --force", sudo=True, expected_exit_code=0)
 
-    def get_configuration(self) -> Dict[str, str]:
+    def get_configuration(self, force_run: bool = False) -> Dict[str, str]:
         if isinstance(self.node.os, CoreOs):
             waagent_conf_file = "/usr/share/oem/waagent.conf"
         else:
             waagent_conf_file = "/etc/waagent.conf"
 
         config = {}
-        cfg = self.node.tools[Cat].run(waagent_conf_file).stdout
+        cfg = self.node.tools[Cat].run(waagent_conf_file, force_run=force_run).stdout
         for line in cfg.splitlines():
             matched = self._key_value_regex.fullmatch(line)
             if matched:
                 config[matched.group("key")] = matched.group("value")
 
         return config
 
-    def get_root_device_timeout(self) -> int:
-        waagent_configuration = self.get_configuration()
+    def get_root_device_timeout(self, force_run: bool = False) -> int:
```

Comment:
Those methods are not changed this time, let's keep them unchanged.

Reply:
updated